app/decorators/models/variant/add_relation_methods.rb

```diff
@@ -0,0 +1,23 @@
+module SolidusRelatedProducts
+  module Variant
+    module AddRelationMethods
+      def name_for_relation
+        descriptive_name
+      end
+
+      def backend_path_for_relation
+        Spree::Core::Engine.routes.url_helpers.edit_admin_product_variant_path(product, self)
```

Comment:
I think it's better not to have these routing methods into the models. We can probably use [this helper](https://github.com/solidusio/solidus/blob/bee57ac2cbc1c139f9da48f95ece5fe873a6451e/backend/app/controllers/spree/admin/resource_controller.rb#L248)?